templates/cloudflare/package.json

```diff
@@ -12,32 +12,32 @@
     "typegen": "wrangler types"
   },
   "dependencies": {
-    "@remix-run/cloudflare": "*",
-    "@remix-run/cloudflare-pages": "*",
-    "@remix-run/react": "*",
-    "isbot": "^4.1.0",
-    "miniflare": "^3.20231030.4",
+    "@remix-run/cloudflare": "^2.8.1",
+    "@remix-run/cloudflare-pages": "^2.8.1",
+    "@remix-run/react": "^2.8.1",
```

Comment:
We set these Remix deps to `*` on purpose so that they always take the latest version of Remix upon app creation - could you revert that change?